definitions/npm/react-dom_v18.x.x/flow_v0.127.x-/react-dom_v18.x.x.js

```diff
@@ -22,6 +22,27 @@ declare module 'react-dom' {
     container: Element
   ): React$Portal;
 
+  declare type CreateRootOptions = {
```

Comment:
Doesn't this need to be in `react-dom/client`? Docs say so [here](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis)